File: PHP/1017.php
<?php
// Problema: 1017 - Gasto de Combustível | Resposta: Accepted
// Linguagem: PHP (8.1.2) [+1s]          | Tempo: 0.003s

const C = 12;
$h = intval(trim(fgets(STDIN)));
$vm = intval(trim(fgets(STDIN)));
printf("%.3f\n", ($h*$vm)/C);
?>

File: Java/1017.java
// Problema: 1017 - Gasto de Combustível | Resposta: Accepted
// Linguagem: Java 19 [+2s]              | Tempo: 0.089s

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
	
	static int pegaInt(BufferedReader in) throws IOException {
		return Integer.parseInt(in.readLine());
	}
	
	static double consumo = 12.0;

    public static void main(String[] args) throws IOException {
    	InputStreamReader ir = new InputStreamReader(System.in);
	    BufferedReader in = new BufferedReader(ir);
	    
	    int 
	    	tempo = pegaInt(in),
	    	velocidade = pegaInt(in);

        System.out.printf("%.3f\n", (tempo*velocidade)/consumo);
        
        in.close();
        ir.close();
    }
}